src/ui/mod.rs

```diff
@@ -0,0 +1,28 @@
+//! Collection of UI helpers.
+
+mod mouse_hover;
+mod widgets;
+
+use bevy::{color::palettes::tailwind, prelude::*};
+
+pub use widgets::*;
+
+const BUTTON_NORMAL: Color = Color::Srgba(tailwind::GRAY_800);
+const BUTTON_HOVER: Color = Color::Srgba(tailwind::GRAY_600);
+const BUTTON_PRESSED: Color = Color::Srgba(tailwind::GRAY_400);
+
+const BUTTON_TEXT: Color = Color::Srgba(tailwind::GRAY_100);
+const LABEL_TEXT: Color = Color::Srgba(tailwind::AMBER_300);
+
+const BACKGROUND: Color = Color::Srgba(tailwind::GRAY_900);
```

Comment:
For a future PR, it would be nice to define these in a hot-reloadable config asset instead of hardcoding.

Reply:
Not sure how much boilerplate that requires without `bevy_common_assets`. Whatever we do, it would have to be completely scrapped once BSN lands, so maybe it's best to just wait for that.